src/products/firefox_android/testing.md

```diff
@@ -1,34 +1,29 @@
+Once you have localized [all the strings needed to complete the Firefox for Android project](how_to_localize.md), and your language is now up in [Nightly builds](https://play.google.com/store/apps/details?id=org.mozilla.fenix), you should start testing! This is the moment to engage as many people as possible: have them use and check out these builds regularly. The more eyes on them, the better.
 
 ### What to look out for?
 
+(For more detailed explanations about testing, please refer to [testing localization of Mozilla projects](l10n_testing.md))
+
 * Language quality.
-* Truncated words (cut-off from screen).
+* Truncated words (cut-off from screen) or overlaps.
 * Anything that appears broken on the UI.
 * Check out at all the main screens, UI, menus, tabs, new features, etc. Make sure these all look good, that everything is properly translated and appears as expected.
 * Font support.
 * Check that all searchplugins are working correctly.
 
-It is strongly suggested to start using these builds on a daily basis, so that you can get into as many screens as possible and check that everything looks fine.
-
-It is important that the people testing the localization be not only the one(s) who have localized. Ask other people from your community to help test in order to have fresh eyes looking at your work.
-
-## Testing on multi-locale builds (shipping in Play Store)
+If you find issues such as the ones listed above, there are two possible options: fix it by yourself directly via Pontoon, or get a mobile developer to fix it. It all depends what kind of issue you find.
 
-Once the Fennec Project Manager (currently **delphine** at **mozilla** dot **com**) sees that you’ve done at least one cycle on a single-locale build, that you’ve continued keeping your work up to date, and that you have been TESTING it - l10n-drivers will consider adding your language to the multi-locale builds. This means that - congratulations! - your locale will now officially be shipping on Firefox for Android.
+In fact, language quality, truncations, overlaps and unlocalized strings can usually be fixed directly via Pontoon. You can verify that your fix has worked by checking it in the new Nightly version that usually ships the next day.
 
-Once your locale is added to the multi-locale builds, it will “ride the trains” like for Firefox Desktop. It will first start shipping on Nightly on the Play Store, then once next merge day arrives it will ship on Beta, and again the following merge day, it will arrive on Release (cycles are usually 6 or 8 weeks). You can check all upcoming merge days on the [Rapid Release Calendar](https://wiki.mozilla.org/RapidRelease/Calendar).
+If the issue is something else, you will probably have to get a mobile developer’s help to fix it. Please [file an issue here](https://github.com/mozilla-mobile/fenix/issues/new/choose), and CC the mobile PM in charge (GitHub name is current “delphine”).
```

Comment:
I'm not sure a lot of people are familiar with CCing someone to an issue, so I would explicitly mention the use of "@ followed by the GitHub username".